Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -166,7 +166,8 @@ export class LGServer {
         const lgTextFile = lgTextFiles.find((item) => item.id === fileId);
         if (lgTextFile) {
           const lgFile = lgUtil.parse(lgTextFile.id, lgTextFile.content, lgTextFiles);
-          return lgUtil.updateTemplate(lgFile, templateId, { body: content });
```

Comment:
lgUtil.updateTemplate now accept `resolver` and return `lgFile`, here miss `resolver` then diagnostic show imported file not find